sources/common/utils.go

```diff
@@ -172,7 +172,7 @@ func RunParallelTasks[I any, O any](input []I, numWorkers int, f func(i I, mutex
 	}()
 
 	mutex := &sync.Mutex{}
-	logger.Log.Debug(fmt.Sprint("initiating workers", numWorkers))
+	logger.Log.Debug(fmt.Sprint("initiating workers = ", numWorkers))
```

Comment:
nit: having special characters in logs makes it hard to parse or apply regexes

Reply:
Changed to a sentence